### DISCUSSION TITLE
compatibility with Java17

### DISCUSSION THREAD

**Discussion by akrattan** [qos-ch/slf4j#347] [General]
Hello,

Is SLF4J version 2.0.7 compatible with Java17?

Regards,
Abhishek

**Comment by hazendaz** [comment]
Yes.  So is 1.7.36

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: akrattan ***@***.***>
Sent: Tuesday, June 6, 2023 11:16:45 PM
To: qos-ch/slf4j ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [qos-ch/slf4j] compatibility with Java17 (Discussion #347)


Hello,

Is SLF4J version 2.0.7 compatible with Java17?

Regards,
Abhishek
